### PR TITLE
fix(ui): alert wizard polish

### PR DESCRIPTION
**Pull request by robinrendle**
- Added alert wizard illustrations
- Tidied up design for alert wizard steps
- Improved visual design for metric and issue alerts
- Added max-width to alert wizard so it's easier to navigate 

## Before
CleanShot 2021-04-16 at 19 21 30@2x

CleanShot 2021-04-16 at 19 20 17@2x

CleanShot 2021-04-16 at 19 20 55@2x

### CODE REVIEW

File: static/app/views/settings/projectAlerts/create.tsx [hunk 1]
```diff
@@ -120,48 +118,58 @@ class Create extends React.Component<Props, State> {
...
+        </AlertConditionsBody>
       </React.Fragment>
     );
   }
 }
 
-const StyledPageHeader = styled(PageHeader)`
-  margin-bottom: ${space(4)};
+const AlertConditionsBody = styled(Layout.Body)`
+  margin-bottom: -20px;
```
Comment (davidenwang) [inline static/app/views/settings/projectAlerts/create.tsx:168]: can we also do `-${space(3)}` here as well?

File: static/app/views/alerts/wizard/options.tsx [hunk 2]
```diff
@@ -1,3 +1,15 @@
+import diagramApdex from 'sentry-images/spot/alerts-wizard-apdex.svg';
```
Comment (davidenwang) [inline static/app/views/alerts/wizard/options.tsx:1]: great job on the diagrams ✨

**Review by davidenwang — APPROVED** [review]
good stuff thanks for all the polish work

File: static/app/views/alerts/wizard/index.tsx [hunk 3]
```diff
@@ -94,83 +90,94 @@ class AlertWizard extends React.Component<Props, State> {
...
+            <Layout.Main fullWidth>
+              <List symbol="colored-numeric">
+                <StyledListItem>{t('What should we alert you about?')}</StyledListItem>
+                <WizardBody>
+                  <WizardOptions>
+                    <Styledh2>{t('Errors')}</Styledh2>
+                    {AlertWizardOptions.map(({categoryHeading, options}, i) => (
+                      <OptionsWrapper key={categoryHeading}>
+                        {i > 0 && <Styledh2>{categoryHeading}</Styledh2>}
+                        <RadioPanelGroup
```
[the remark this answers is not in the document]
Comment (robinrendle) [reply static/app/views/alerts/wizard/index.tsx:116]: Ah that would be great, thanks.

File: static/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx [hunk 4]
```diff
@@ -249,108 +248,100 @@ class ActionsPanel extends React.PureComponent<Props> {
...
-                        onChange={this.handleChangeActionLevel.bind(
-                          this,
-                          triggerIndex,
-                          actionIdx
-                        )}
-                        value={triggerIndex}
-                        options={levels}
-                      />
+      <React.Fragment>
+        <StyledListItem>{t('Perform actions')}</StyledListItem>
```
[the remark this answers is not in the document]
Comment (robinrendle) [reply static/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx:252]: v good point, done and done

**Pull request merged**